key,default_value
userAgent,"Botocore/1.16.26 Python/2.7.18 Linux/5.4.0-2-amd64"
eventID,"832649-863e-4d55-ac72-8f51f7906f2a"
errorMessage,"User: arn:aws:iam::811596193553:user/backup is not authorized to perform: iotsitewise:ListAssets on resource: *"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDA9BO36HFBHKGJAO9C1"
userIdentity_arn,"arn:aws:iam::811596193553:user/backup"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA01U43UX3RBRDXF4Q"
userIdentity_userName,"backup"
eventType,"AwsApiCall"
errorCode,"AccessDenied"
sourceIPAddress,"251.105.254.1"
eventName,"ListAssets"
eventSource,"iotsitewise.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters,"None"
awsRegion,"us-east-1"
requestID,"4f1bbd74-4fe7-4b51-882b-3c4d48a07ec3"
responseElements,"None"
eventVersion,"1.05"
eventTime,"2020-06-10T05:31:11Z"
